modules/@apostrophecms/page-type/index.js

```diff
@@ -323,7 +323,14 @@ module.exports = {
           return self.apos.page.insert(_req, doc.aposLastTargetId.replace(':draft', ':published'), doc.aposLastPosition, published, options);
         } else if (!doc.level) {
           // Insert the home page
-          return self.apos.doc.db.insertOne(_req, published, options);
+          Object.assign(published, {
+            path: doc.path,
```

Comment:
Why this change?

Reply:
`published` at this point only has schema properties. If we don't bring over this information the home page will not replicate properly. It's different for non-home pages because we can use `page.insert` for them.